puppet/manifests/base.pp

```diff
@@ -8,9 +8,13 @@
   'npm': ensure => "present";
   'couchdb': ensure => "present";
   'libffi-dev': ensure => "present";
+  'libsqlite3-dev': ensure => "present";
   } ->
   class { 'ruby_install': 
     ruby_version => "2.2.1",
+  } ->
+  exec { 'mail-catcher':
+    command => 'gem install mailcatcher'
```

Comment:
It strikes me as odd that this is the only `gem install`. Why is mailcatcher special?